js/client_exporter/xlsx/xlsx_elements.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * @typedef {string} Enums.XlsxUnderlineType
+ * @enum {'double'|'doubleAccounting'|'none'|'single'|'singleAccounting'}
```

Comment:
Can we move enums to the [js/docEnums.js](https://github.com/DevExpress/DevExtreme/blob/18_2/js/docEnums.js)